packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/alb/application-listener.ts

```diff
@@ -188,12 +249,35 @@ export class ApplicationListener extends BaseListener implements IApplicationLis
       throw new Error('At least one of \'port\' or \'protocol\' is required');
     }
 
+    if (props.mutualAuthentication) {
+      const currentMode = props.mutualAuthentication.mutualAuthenticationMode;
+
+      if (currentMode === MutualAuthenticationMode.VERIFY && !props.mutualAuthentication.trustStore) {
+        throw new Error(`You must set 'trustStore' when 'mode' is '${MutualAuthenticationMode.VERIFY}'`);
+      }
+
+      if ((currentMode === MutualAuthenticationMode.OFF || currentMode === MutualAuthenticationMode.PASS_THROUGH)) {
```

Comment:
nit: I think the double brackets here might be unnecessary?
```suggestion
      if (currentMode === MutualAuthenticationMode.OFF || currentMode === MutualAuthenticationMode.PASS_THROUGH) {
```